src/parameter/parametervirtualwidget.cpp

```diff
@@ -11,6 +12,24 @@ ParameterVirtualWidget::ParameterVirtualWidget(QWidget *parent) : QWidget(parent
 	policy.setHorizontalStretch(0);
 	policy.setVerticalStretch(0);
 	this->setSizePolicy(policy);
+
+	this->object = parameterobject;
```

Comment:
As this is a pure assignment, we can clarify this by instead adding it to the initialization list for this constructor.